zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -29,6 +29,9 @@ zipkin:
     lookback: ${QUERY_LOOKBACK:86400000}
     # The Cache-Control max-age (seconds) for /api/v1/services and /api/v1/spans
     names-max-age: 300
+    # CORS allowed-origins.
+    allowed-origins: "*"
```

Comment:
ah ok. I just saw `.*` used later in the file, so figured it might be valid to use `*`